lib/commands/setup.js

```diff
@@ -1391,7 +1392,11 @@ SetupScreens.prototype.sdk = function sdk(callback) {
 			ignoreFiles: new RegExp(this._config.get('cli.ignoreFiles')),
 			validate: function (value) {
 				if (value && (!fs.existsSync(afs.resolvePath(value) || fs.statSync(value).isDirectory()))) {
```

Comment:
Also, though not coming from your changes - this logic doesn't sound right to me.

We should resolve the path to verify whether it exists and then, check whether the resolved path is a directory or not. Looks we are not doing it here. So, if we provide a path to a file, then we might end up saving the path to the file. The code should be something like

```
if (value && !(fs.existsSync(afs.resolvePath(value)) && fs.statSync(value).isDirectory())) {
```